docs/user/contributing/getting-involved.md

```diff
@@ -41,12 +41,12 @@ Alongside the forums, you can communicate with developers or others in the commu
 
 ## Funding
 
```

Comment:
The numbered list in this section is now redundant and can be removed; we changed the wording on our OC page to cover the "what we spend money on" topic. In my mind, having it in two places means we have to remember to keep them in sync.

Do you want to take a stab at adding a sentence or two that says "see our OC page for details"?

Reply:
Good point; I've added a first version of such an explanatory sentence and removed the list.